.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on: [push]
```

Comment:
This is entirely up to you, but a structure that I have found useful for triggering CI builds is more like this:

```
on:
  push:
    branches:
      # Run CI on the main branch whenever a PR is merged
      - main
      # If you support maintenance branches, do the same on those branches
      - my-maintenance-branch-name-[0-9]+.[0-9]+.x
  pull_request:
    branches:
      - main
```

The current structure (if I'm not mistaken) will not run CI if there is a PR from a remote fork.

Reply:
Following up in #353.